examples/cockroachdb/cockroachdb-statefulset.yaml

```diff
@@ -66,6 +78,21 @@ spec:
       labels:
         app: cockroachdb
       annotations:
+        scheduler.alpha.kubernetes.io/affinity: >
```

Comment:
Based on the [design proposal](https://github.com/kubernetes/kubernetes/blob/9ac1bcaa0337e8a54554c894a07366ae0eae52e4/docs/design/podaffinity.md), `requiredDuringSchedulingRequiredDuringExecution` should avoid from scheduling the pod. Can you open a bug about it?